pycqed/measurement/quantum_experiment.py

```diff
@@ -661,5 +662,74 @@ def save_timers(self, quantum_experiment=True, sequence=True, segments=True, fil
             raise e
 
 
+    def plot(self, sequences=None, segments=None, qubits=None,
+             save=False, **plot_kwargs):
+        """
+        Plots (a subset of) sequences / segments of the QuantumExperiment
+        :param sequences (int, list): sequences to plot. Can be None (plot all
+        sequences), an integer (index of sequence to plot),  or a list of
+        integers/str. If strings are in the list, then plots only sequences
+        with the corresponding name.
+        :param segments (int, list): index of segments to plot. Plots all
+        segments by default.
+        :param qubits (list): list of qubits to plot
+        :param save (bool): save the figures.
+        :param plot_kwargs: kwargs passed on to segment.plot()
+        :return:
+        """
+        plot_kwargs = deepcopy(plot_kwargs)
+        if sequences is None:
+            # plot all sequences
+            sequences = self.sequences
+        elif isinstance(sequences, int):
+            sequences = [self.sequences[sequences]]
+        elif isinstance(sequences, list):
+            if len(sequences) and isinstance(sequences[0], int):
+                # sequences is a list of indices of sequences to plot
+                sequences = [self.sequences[s] for s in sequences]
+            elif len(sequences) and isinstance(sequences[0], str):
+                # sequences is a list of names of sequences to plot
+                sequences = [[s for s in self.sequences if s.name == seqname]
+                             for seqname in  sequences]
+                sequences = np.ravel(sequences).tolist()
+        if qubits is None:
+            qubits = self.meas_objs
+        plot_kwargs.update(dict(channel_map=plot_kwargs.pop('channel_map',
+                           general.get_channel_map(qubits))))
```

Comment:
First comment would be addressed by cherry-picking as you suggested (yes, it would be OK to cherry-pick), but what about the second comment? Also note that the function you intend to cherry-pick has no docstring.